### ISSUE TITLE
[TW-1038] Config::load called twice

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#1073]
_Paul Beckingham on 2009-12-13T17:02:43Z says:_ 

 It appears that Config::load is called twice, no matter what.  Why did I do that?

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2009-12-13T17:02:43Z
Modified: 2014-02-09T02:15:15Z
```

**Comment by taskwarrior** [comment]
_Federico Hernandez on 2010-01-22T12:34:19Z says:_ 

 To be checked again in 2.0.x.

Has been addressed in 1.9.0 with commenting out the corresponding lines.

**Comment by taskwarrior** [comment]
_Federico Hernandez on 2010-05-30T16:28:57Z says:_ 

 Closing it now as no problems have emerged since the change.
